broker/configuration.go

```diff
@@ -24,6 +24,7 @@ import (
 type Configuration struct {
 	Address      string     `mapstructure:"address" toml:"address"`
 	Topic        string     `mapstructure:"topic" toml:"topic"`
+	PublishTopic string     `mapstructure:"publish_topic" toml:"publish_topic"`
```

Comment:
Why not to use `Topic`?

Reply:
When we need to use the `producer` package probably we want to use a different topic for consuming and producing messages